charts/third-party/authup/templates/deployment.yaml

```
@@ -47,6 +47,20 @@ spec:
+            {{- $additionalDomains := .Values.additionalDomains }}
+            {{- if kindIs "slice" $additionalDomains }}
+              {{- $additionalDomains = join "," $additionalDomains }}
+            {{- end }}
+            {{- if not $additionalDomains }}
+              {{- $flameHub := dig "flameHub" dict (.Values.global | default dict) }}
+              {{- if or (dig "ingress" "enabled" false $flameHub) (dig "gatewayApi" "enabled" false $flameHub) }}
+                {{- $additionalDomains = include "flameHub.uiDomain" . }}
+              {{- end }}
+            {{- end }}
+            {{- with $additionalDomains }}
+              -   name: ADDITIONAL_DOMAINS
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)